Kicked failed test. Didn't recognize it as an intermittent failure, but also seems unlikely to be caused by this PR.
```
FAIL: casexml.apps.phone.tests.test_sync_mode:SyncTokenUpdateTestSQL.testArchiveUpdates
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 430, in testArchiveUpdates
    self.assertEqual(sync.cases, {})
AssertionError: {'archive_syncs': <casexml.apps.case.mock.case_block.CaseBlock object at 0x7f46c [truncated]... != {}
- {'archive_syncs': <casexml.apps.case.mock.case_block.CaseBlock object at 0x7f46c1b45950>}
+ {}
```

--- a/corehq/form_processor/management/commands/run_sql.py
+++ b/corehq/form_processor/management/commands/run_sql.py
@@ -84,7 +84,7 @@ def run_once(sql, dbname):
 class RunUntilZero(object):
     """SQL statement to be run repeatedly
 
-    ...until the first column of of the first returned row is zero.
+    ...until the first column of the first returned row is zero.
     """
 
     sql = attr.ib()

